CMakeLists.txt

```
@@ -112,6 +112,7 @@ OPTION(KWIVER_ENABLE_SPROKIT              "Enable building sprokit" OFF )
 if(KWIVER_ENABLE_ARROWS)
   OPTION(KWIVER_ENABLE_MVG                "Enable Multi-View Geometry Arrow" ON )
   OPTION(KWIVER_ENABLE_KLV                "Enable Key-Length-Value Metadata Arrow" ON)
+  OPTION(KWIVER_ENABLE_STANAG             "Enable NATO Standardization Agreement Arrow" ON)
```

There are a bunch of STANAGs, of which 4607 is just one (and 4676 is probably also interesting, along with 4609). Perhaps
```suggestion
  OPTION(KWIVER_ENABLE_4607             "Enable NATO Standardization Agreement 4607 (GMTI) Arrow" ON)
```
and corresponding changes below.

---

@judajake Do you have a preference between making each STANAG its own arrow or having one arrow for all of them?

---

My bias is towards having one arrow.  Longterm, hopefully, more will be supported.

---

I think you already have two (since that is what the KLV arrow is - STANAG 4609). So it would not be consistent to say that this is *the* NATO Standardization Agreement Arrow.